.travis.yml

```diff
@@ -13,6 +13,7 @@ install: yarn --frozen-lockfile
 
 script:
   - yarn lint
+  - yarn format
```

Comment:
@jzempel What prompted this change?  

I would be concerned if Travis was testing and deploying code that was transformed by a `format` without it being visible in Github.

Reply:
We seem to continue to let non-formatted code slip through (misconfigured husky?). The travis build will fail if the git tree isn't clean after test.

Reply:
Oh that's perfect 💯